terraform/modules/azure/storage.tf

```diff
@@ -0,0 +1,33 @@
+resource "azurerm_storage_account" "tempdata" {
+  name                              = var.tempdata_storage_account_name
+  resource_group_name               = var.resource_group_name
+  location                          = var.region_name
+  account_replication_type          = var.environment_name != "production" ? "LRS" : "RAGRS"
+  account_tier                      = "Standard"
+  access_tier                       = "Hot"
+  account_kind                      = "StorageV2"
+  min_tls_version                   = "TLS1_2"
```

Comment:
Apart from `allow_nested_items_to_be_public` all these properties are set to the default version as per the [docs](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_account).  Do we need to explicitly set them?

Reply:
Tidied up. I added them in for a reason - not sure why now :)